<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DControlBolasTipos</class>
 <widget class="QDialog" name="DControlBolasTipos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>640</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabBolas">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>301</width>
     <height>481</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="btTipo1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>89</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btTipo2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>590</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btTipo3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>590</y>
     <width>89</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tipo de las pelotas</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTipo">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btImpactos">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Impactos Lista</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btImpactosTabla">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>540</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Impactos Tabla</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DControlBolasTipos</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DControlBolasTipos</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
